frontend/src/metabase/admin/datamodel/components/database/MetadataHeader.jsx

```diff
@@ -15,7 +15,7 @@ export default class MetadataHeader extends Component {
         databases: PropTypes.array.isRequired,
         selectDatabase: PropTypes.func.isRequired,
         isShowingSchema: PropTypes.bool.isRequired,
-        toggleShowSchema: PropTypes.func.isRequired,
+        toggleShowSchema: PropTypes.func.isRequired
```

Comment:
it's removing all the hanging commas!

Reply:
Dura Lex Sed Lex

Reply:
we could enable this option https://prettier.io/docs/en/options.html#trailing-commas